### PR TITLE
Feature/react ga4

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

This PR adds `react-ga4`

It seems to be working just from the demo
Screenshot 2023-02-02 at 16 05 33

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1,2 @@
-DUNE_API_KEY=tuBkOcda6ymMvmnpIX9Qer9Xb5hJ2rf8
\ No newline at end of file
+DUNE_API_KEY=tuBkOcda6ymMvmnpIX9Qer9Xb5hJ2rf8
+NEXT_PUBLIC_GOOGLE_ANALYTICS=
```
Comment (alfetopito) [inline .env:2]: I checked and this env var is already configured on vercel.

The question then is, why add it now to the env empty?

**Comment by elena-zh** [comment]
Hey @nenadV91 , how can I test it?

File: .env (hunk 1 above)
Comment (nenadV91) [reply .env:2]: Its empty here just to show users that it is there and how to define it

**Review by anxolin — APPROVED** [review]
Approving judging on the code changes. However, this should be tested in analytics. 

Also, validate with @MindyCoW and Alex Marsh / @avsavsavs the codes they want us to add for each project analytics ID.

**Review by alfetopito — APPROVED** [review]

**Review by shoom3301 — APPROVED** [review]

**Pull request merged**